canonical/ubuntu.com
# feat: /iot redesign

## Done

- Applied changes as per [doc](https://docs.google.com/document/d/1nBx1SP3Y10gXX0rZPDaz1Gudy-hWP4XRMUcETyPBldI/edit?tab=t.0#heading=h.kbutqxr7kac7) and [design](https://www.figma.com/design/1CdIT6TV3ASf55GsnQcwXg/Internet-of-Things?node-id=1-9092&node-type=instance&m=dev)

## QA

- View the site locally in your web browser at: https://ubuntu-com-14537.demos.haus/iot
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-16628
